android/Gutenberg/src/main/java/org/wordpress/gutenberg/GutenbergView.kt

```diff
@@ -41,8 +41,8 @@ class GutenbergView : WebView {
     private var siteApiRoot: String = ""
     private var siteApiNamespace: String = ""
     private var authHeader: String = ""
-    private var lastUpdatedTitle = ""
-    private var lastUpdatedContent = ""
+    private var lastUpdatedTitle: CharSequence? = null
```

Comment:
Using `CharSequence` matches the WP-Android implementation for other editors. Using a nullable valuable simplifies conditional logic within `getTitleAndContent`.